src/dataset.jl

```diff
@@ -44,7 +46,8 @@ Overwrite `joinpath(dataset.path, "recordings.msgpack.zst")` with the contents
 of `dataset.recordings`.
 """
 function save_recordings_file(dataset::Dataset)
```

Comment:
It seems like there's a disconnect in terminology here. I think either this function should be renamed as well or `read_recordings_file` and `write_recordings_file` should just be exported as-is without a rename. Tbh I'd prefer the latter, as the fact that the recordings metadata file is stored as a zstd-compressed, msgpack formatted file seems like an implementation detail to me.

Reply:
Hmmm...funnily enough, I actually changed those names away from `read_recordings_file`/`write_recordings_file` in order to separate them further from the `Dataset` API. Might've made it more confusing though I guess 😅 

I suppose I have a notion of package/interface structure here that's not really explicit...IMO, Onda.jl has a lower-level "map-exactly-to-Onda-metadata-specification" API (located in "Onda Metadata Objects" in the docs and in the poorly-named `recordings.jl` in the source) and some higher-level convenience APIs that are implemented on top (e.g. the `Dataset` API). The `save_recordings_file` function is part of the higher-level `Dataset` API, whereas `read_recordings_msgpack_zst` and `write_recordings_msgpack_zst` are part of the lower-level API (and have names that match exactly to the spec for that reason).

I'm open to changing the names though if these still seem bad or if we can identify some clearly better names

Reply:
Ah, okay, thanks for the explanation. I didn't realize that there was any real distinction in the dataset and lower-level APIs. I guess it makes sense to use these names then.